scripts/release/createWebModules.js

```diff
@@ -126,8 +135,41 @@ async function commonActions(moduleInfo, widgets) {
         );
     }
     await commitAndCreatePullRequest(moduleInfo);
-    await updateTestProjectWithWidgetsAndAtlas(moduleInfo, tmpFolder, widgets);
+    await updateTestProjectWithJavascript(tmpFolder, moduleInfo);
+    const mpkOutput = await createMPK(tmpFolder, moduleInfo, regex.excludeFiles);
+
+    await createGithubReleaseFrom({
+        title: `${moduleInfo.nameWithSpace} - Marketplace Release v${moduleInfo.version}`,
+        body: moduleChangelogs.replace(/"/g, "'"),
+        tag: `${moduleFolderNameInRepo}-v${moduleInfo.version}`,
+        mpkOutput,
+        isDraft: true
+    });
+    await execShellCommand(`rm -rf ${tmpFolder}`);
+
+    console.log("Done.");
+}
+
+async function commonActions(moduleInfo, widgets = [], exportWithoutSnippet = false, updateWidgetsChangelogs = false) {
```

Comment:
For the 2 new arguments we have default values for backwards compatibility, but these names aren't the most intuitive imo. Like `updateWidgetsChangelogs` true means that the widget changelogs should be updated, but it being false means to _not only_ not update the widget changelogs but to _also_ update the module changelogs. This can be a bit confusing

Is there every a scenario where there will be a 3rd option? If so, it might be better to make this an enum of some kind like `changelogScope: 'MODULE' | 'WIDGET'`. Not sure if it's realistic, but just an idea.

Reply:
done